src/WinRT.Runtime/Projections/NotifyCollectionChangedEventHandler.cs

```diff
@@ -41,28 +42,32 @@ static unsafe NotifyCollectionChangedEventHandler()
             var nativeVftbl = ComWrappersSupport.AllocateVtableMemory(typeof(NotifyCollectionChangedEventHandler), sizeof(global::WinRT.Interop.IDelegateVftbl));
             Marshal.StructureToPtr(AbiToProjectionVftable, nativeVftbl, false);
             AbiToProjectionVftablePtr = nativeVftbl;
+
+            IID = Projections.UiXamlModeSetting == Projections.UiXamlMode.WindowsUiXaml
```

Comment:
Nit: this should also just be a switch with two RVA fields, rather than parsing at runtime